src/Amazon.php

```diff
@@ -74,11 +82,17 @@ class Amazon
      * @param  string $countryCode Country code for Amazon service; may be US, UK, DE, JP, FR, CA
      * @param  string $secretKey   API Secret Key
      * @param  string $version     API Version to use
+     * @param  bool   $useSsl      Use HTTPS instead of HTTP?
      * @throws Exception\InvalidArgumentException
      * @return Amazon
      */
-    public function __construct($appId, $countryCode = 'US', $secretKey = null, $version = null)
-    {
+    public function __construct(
+        $appId,
+        $countryCode = 'US',
+        $secretKey = null,
+        $version = null,
+        $useSsl = false
```

Comment:
I see that Travis build 169 passed when it should have failed. This is puzzling; when I check out commit 619efaa and run tests, they fail as they should. Perhaps I made my second fix so quickly that the Travis build ran against the newer code both times.